moto/s3/responses.py

```diff
@@ -849,26 +849,33 @@ def _get_path(request):
     def _bucket_response_delete_keys(self, request, body, bucket_name):
         template = self.response_template(S3_DELETE_KEYS_RESPONSE)
 
-        keys = minidom.parseString(body).getElementsByTagName("Key")
-        deleted_names = []
+        objects = minidom.parseString(body).getElementsByTagName("Object")
+
+        deleted_objects = []
         error_names = []
-        if len(keys) == 0:
+        if len(objects) == 0:
             raise MalformedXML()
 
-        for k in keys:
-            key_name = k.firstChild.nodeValue
+        for object_ in objects:
+            key_name = object_.getElementsByTagName("Key")[0].firstChild.nodeValue
+            version_id_node = object_.getElementsByTagName("VersionId")
+            if version_id_node:
+                version_id = version_id_node[0].firstChild.nodeValue
+            else:
+                version_id = None
```

Comment:
Not really an expert on manipulating XML DOM, there may be a more elegant way to do this

Reply:
Hi @antoinewdg-mt, great to see you here :). Did you take a look at [etree](https://docs.python.org/3.8/library/xml.etree.elementtree.html) ? It think it's possible to create a xpath lookup for your problem. Hope it helps :)

Reply:
In other parts of moto we tend to use xmltodict:
```xmltodict.parse(body)["Object"]["Key"]["VersionId"]```

Let's use that here as well - it's quite clean, and it makes it obvious what the expected input is